src/components/Lane.js

```diff
@@ -183,7 +183,7 @@ class Lane extends Component {
           {...card}
         />
       )
-      return draggable && cardDraggable ? (
+      return draggable && cardDraggable && !card.notDraggable ? (
```

Comment:
I think it would be better to make this `draggable`, to match other boolean values, with it having a default value of `true`.

That way it better matches the rest of the API (such as lanes having `droppable`, not `notDroppable`.

Reply:
done